.github/workflows/ci_python.yml

```diff
@@ -0,0 +1,175 @@
+# This file is autogenerated by maturin v1.2.0
+# To update, run
+#
+#    maturin generate-ci github --pytest --zig -o ../.github/workflows/ci_python.yml
+#
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: [x86_64, x86, aarch64, armv7, s390x, ppc64le]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+      - name: Build wheels
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.target }}
+          args: --release --out dist --zig
+          sccache: 'true'
+          manylinux: auto
+          working-directory: ${{ github.workspace }}/python
+      - name: Upload wheels
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheels
+          path: ${{ github.workspace }}/python/dist
+      - name: pytest
+        if: ${{ startsWith(matrix.target, 'x86_64') }}
+        shell: bash
+        working-directory: ${{ github.workspace }}/python
+        run: |
+          set -e
+          pip install kaskada --find-links dist --force-reinstall
```

Comment:
I need to verify -- the intention is to build the wheel, and then install using that, which shouldn't require any building and should let us test things.